haystack/nodes/preprocessor/preprocessor.py

```diff
@@ -11,15 +11,14 @@
 
 from tqdm import tqdm
 from more_itertools import windowed
+from transformers import PreTrainedTokenizerBase
```

Comment:
You can't just import from transformers anymore: it's now an optional dependency. See, for example, how the retrievers handle it: https://github.com/deepset-ai/haystack/blob/adfabdd6483b9d145732e2739784049f619648af/haystack/nodes/retriever/dense.py#L29

Reply:
So it would just be:

```python
with LazyImport("Run 'pip install transformers'") as torch_and_transformers_import:
    from transformers import PreTrainedTokenizerBase
```

instead of the current import?